xml/System.Reflection.PortableExecutable/DebugDirectoryEntryType.xml

```diff
@@ -68,6 +68,15 @@ The debug directory entry of type <xref:System.Reflection.PortableExecutable.Deb
       <MemberValue>2</MemberValue>
       <Docs>
         <summary>Associated PDB file description.</summary>
+        <remarks>
```

Comment:
This shouldn't be added because

1. It won't show because these are enum fields so they don't have remarks
2. This info is already part of the class remarks

Reply:
Ok. Removing.